glib/src/variant.rs

```diff
@@ -290,6 +300,28 @@ impl Variant {
         }
     }
 
+    /// Tries to extract a `&[u8]` from a variant of type `ay` (array of bytes).
+    ///
+    /// Returns an error if the type is not `ay`.
+    pub fn bytes(&self) -> Result<&[u8], VariantTypeMismatchError> {
```

Comment:
Should really update all the other `Option`s to `Result`s :)

Reply:
Yeah, I'd just also like to use this stuff on 0.14 so will try to do a larger pass for "0.15 API changes" at some point later.